bootstrap/templates/ansible/playbooks/cluster-rollout-update.yaml.j2

```
@@ -25,24 +25,38 @@
         # If spec.unschedulable is defined then the node is cordoned
         - not (kubectl_get_node['stdout'] | from_json).spec.unschedulable is defined
       block:
-        - name: Cordon
-          ansible.builtin.command: "kubectl cordon {% raw %}{{ inventory_hostname }}{% endraw %}"
+        - name: "Cordon {% raw %}{{ inventory_hostname }}{% endraw %}"
```

```suggestion
        - name: Cordon
```

The node name is already included in the ansible output, we don't need it here or the other task names IMO.